packages/ra-core/src/controller/list/useList.spec.tsx

```diff
@@ -62,7 +62,7 @@ describe('<useList />', () => {
                         { id: 2, title: 'world' },
                         { id: 1, title: 'hello' },
                     ],
-                    error: undefined,
```

Comment:
this is to make the output compatible with the ListControllerResult, which is using useQueryResult signature. In it, the empty error is `null`, not `undefined`.